app-admin/fleet/files/fleet.service

```diff
@@ -11,3 +11,6 @@ After=fleet.socket
 ExecStart=/usr/bin/fleetd
 Restart=always
 RestartSec=10s
+
+[Install]
+WantedBy=default.target
```

Comment:
A number of our units do this but I think we should actually be using `multi-user.target` which is what upstream always does.